### PR TITLE
[CLOUDTRUST-4069] Add support endpoint to get email information

### PR DESCRIPTION
**Pull request by fperot74**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -5,8 +5,9 @@ go 1.17
 require (
 	github.com/cloudtrust/common-service/v2 v2.5.6
 	github.com/cloudtrust/httpclient v1.0.7
-	github.com/cloudtrust/keycloak-client/v2 v2.5.4
+	github.com/cloudtrust/keycloak-client/v2 v2.5.5-0.20220215121341-c0ffc58d5ae4
```
Comment (fperot74) [inline go.mod:8]: Use newly released keycloak-client version

**Review by harture — APPROVED** [review]

File: pkg/support/mock_test.go [hunk 2]
```diff
@@ -0,0 +1,6 @@
+package support
+
+//go:generate mockgen --build_flags=--mod=mod -destination=./mock/component.go -package=mock -mock_names=Component=Component,KeycloakClient=KeycloakClient github.com/cloudtrust/keycloak-bridge/pkg/support Component,KeycloakClient
+//xgo:generate mockgen --build_flags=--mod=mod -destination=./mock/authentication_db_reader.go -package=mock -mock_names=AuthorizationDBReader=AuthorizationDBReader github.com/cloudtrust/common-service/v2/security AuthorizationDBReader
```
Comment (basbeu) [inline pkg/support/mock_test.go]: To be removed

File: pkg/support/mock_test.go [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+//xgo:generate mockgen --build_flags=--mod=mod -destination=./mock/kc-auth.go -package=mock -mock_names=KeycloakClient=KcClientAuth github.com/cloudtrust/common-service/v2/security KeycloakClient
```
Comment (basbeu) [inline pkg/support/mock_test.go]: To be removed

File: pkg/support/mock_test.go [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+//xgo:generate mockgen --build_flags=--mod=mod -destination=./mock/dbase.go -package=mock -mock_names=EventsDBModule=EventsDBModule github.com/cloudtrust/common-service/v2/database EventsDBModule
```
Comment (basbeu) [inline pkg/support/mock_test.go]: to be removed

**Review by basbeu — APPROVED** [review]

**Pull request merged**